shell/packages/sandstorm-ui-app-details/app-details-common.js

```diff
@@ -0,0 +1,14 @@
+SandstormAppDetails = function(db, quotaEnforcer, appId) {
```

Comment:
Huh, good point.  I guess in theory the other sandstorm-ui-* probably deserve the same change.